index.js

```diff
@@ -1,8 +1,87 @@
 const battleship = () => {
-  return 'The winner is...?'
+
+  // create objects with the given parametrs
+  let playerOne = {
+    name: "playerOne",
+    shipCount: 0,
+    gameBoard: [
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0]
+    ]
+  }
+  let playerTwo = {
+    name: "playerTwo",
+    shipCount: 0,
+    gameBoard: [
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0], 
+      [0, 0, 0, 0]
+    ]
+  };
+
+  // asking for the players' name
+  playerOne.name = prompt("Player 1 what is your name?");
+  playerTwo.name = prompt("Player 2 what is your name?");
+  
+  // randomly add ships to each board
+  const shipPlace = (player) => {
+    let x = Math.floor(Math.random() *4)
+      let y = Math.floor(Math.random() *4)
```

Comment:
Weird indentation

Reply:
@andresochoa91 you right, didn't notice it.